src/core/client/stream/tabs/Configure/ArchivedConfigurationContainer.tsx

```diff
@@ -0,0 +1,81 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent, useCallback } from "react";
+import { graphql } from "react-relay";
+
+import { useMutation, withFragmentContainer } from "coral-framework/lib/relay";
+import { Icon } from "coral-ui/components/v2";
+import { CallOut } from "coral-ui/components/v3";
+import { Button } from "coral-ui/components/v3/Button/Button";
+
+import { ArchivedConfigurationContainer_story } from "coral-stream/__generated__/ArchivedConfigurationContainer_story.graphql";
+
+import UnarchiveStoriesMutation from "./UnarchiveStoriesMutation";
+
+import styles from "./ArchivedConfigurationContainer.css";
+
+interface Props {
+  story: ArchivedConfigurationContainer_story;
+}
+
+const ArchivedConfigurationContainer: FunctionComponent<Props> = ({
+  story,
+}) => {
+  const unarchiveStories = useMutation(UnarchiveStoriesMutation);
+  const unarchiveStory = useCallback(() => {
+    void unarchiveStories({ storyIDs: [story.id] });
+  }, [story.id, unarchiveStories]);
+
+  return (
+    <div className={styles.root}>
+      <CallOut
+        color="warning"
+        title={
+          <Localized id="configure-archived-title">
+            This comment stream has been archived
+          </Localized>
+        }
```

Comment:
You could make this accessible by:
```suggestion
        color="warning"
        aria-labelledby="configure-archived-title"
        title={
          <Localized id="configure-archived-title">
            <div id="configure-archived-title">
              This comment stream has been archived
            </div>
          </Localized>
        }
```